_data/boards/STM32F401RCT6-STM32F-Core-Board.json

```diff
@@ -38,7 +38,8 @@
         "mounting": "None"
     },
     "remarks": [
-        { "type": "warning", "content": "The +5V pins on this board are directly connected to the +5V pin of the USB connector. There is no protection in place. Do not power this board through USB and an external power supply at the same time." }
+        { "type": "warning", "content": "The +5V pins on this board are directly connected to the +5V pin of the USB connector. There is no protection in place. Do not power this board through USB and an external power supply at the same time." },
+        { "type": "warning", "content": "This board has an external switchable (disabled by default) USB pullup (R11) which is not actually needed (since automatic internal pullups are present) and might actually be harmful (when both pullups are enabled, the USB specification is violated)." }
```

Comment:
Nitpicking, but a little bit clearer warning with shorter sentences and without many of the parentheses:

"The microcontroller on this board features internal pull-up resistors for the USB data lines. However, this board has an additional switchable pull-up resistor (R11), which is disabled by default. This resistor is not needed and violates the USB specification when used with the internal pull-up resistors. This may cause errors while using USB on this board."

And for the non-switchable variant:

"The microcontroller on this board features internal pull-up resistors for the USB data lines. However, this board has an additional pull-up resistor (R...). This resistors is not needed and violates the USB specification when used with the internal pull-up resistors. This may cause errors while using USB on this board."